kafka/producer/base.py

```diff
@@ -111,12 +160,16 @@ def __init__(self, client, async=False,
                  codec=None,
                  batch_send=False,
                  batch_send_every_n=BATCH_SEND_MSG_COUNT,
-                 batch_send_every_t=BATCH_SEND_DEFAULT_INTERVAL):
+                 batch_send_every_t=BATCH_SEND_DEFAULT_INTERVAL,
+                 async_retry_options=ASYNC_RETRY_OPTIONS,
```

Comment:
i think separate kwargs for each option (limit, backoff_ms, and retry_on_timeouts) might be a bit easier for users